RecoParticleFlow/PFProducer/src/PFAlgo.cc

```diff
@@ -2999,7 +3012,8 @@ void PFAlgo::processBlock( const reco::PFBlockRef& blockref,
       reco::PFClusterRef eclusterRef = elements[iEcal].clusterRef();
       assert( !eclusterRef.isNull() );
 
-      double ecalEnergy = eclusterRef->correctedEnergy();
+      // KH: use raw ECAL energy for PF hadron calibration. 
+      double ecalEnergy = eclusterRef->energy(); // ecalEnergy = eclusterRef->correctedEnergy();
```

Comment:
is the commented part of the code needed?
IIUC, there is no single place to make a switch from raw to calibrated.
So, the commented out part here can't be useful as a quick check to switch to the calibrated value.
Please drop the commented out ```// ecalEnergy = eclusterRef->correctedEnergy();``` or add some note why this is still useful here.

Reply:
OK. I will clean up when I touch PFAlgo next time.